community/server/src/main/java/org/neo4j/server/rest/transactional/Neo4jJsonCodec.java

```diff
@@ -28,6 +28,7 @@
 import org.codehaus.jackson.map.ObjectMapper;
 import org.codehaus.jackson.map.SerializationConfig;
 
+import org.neo4j.cypher.internal.compiler.v3_0.ActsAsMap;
```

Comment:
Still depending on 3.0 specific stuff in server, in 3.1 should we cast to a `v3_0.ActsAsMap` or a `v3_1.ActsAsMap`?